src.cmake

```diff
@@ -179,6 +179,7 @@ set(SGAMELIST
     ${GAMELOGIC_DIR}/sgame/components/ThinkingComponent.cpp
     ${GAMELOGIC_DIR}/sgame/components/TrapperComponent.cpp
     ${GAMELOGIC_DIR}/sgame/components/TurretComponent.cpp
+    ${GAMELOGIC_DIR}/sgame/components/SpectatorClassComponent.cpp
```

Comment:
SpectatorClassComponent was being consistent with AlienClass/HumanClass. Do you think spectatorcomponent is better?